igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -297,7 +318,18 @@ class TravisService implements BuildService {
     }
 
     protected String branchFromRepoSlug(String inputRepoSlug) {
-        return inputRepoSlug.tokenize('/').drop(2).join('/') - "pull_request_"
+        String branch = inputRepoSlug.tokenize('/').drop(2).join('/') - ~/^pull_request_/
+        if (branch.equalsIgnoreCase("tags")) {
+            return ""
+        }
+        return branch
+    }
+
+    protected boolean branchIsTagsVirtualBranch(String inputRepoSlug) {
+        return inputRepoSlug.tokenize('/').drop(2).join('/').equalsIgnoreCase("tags")
```

Comment:
You seem to use this `tokeninze('/').drop(2).join('/')` quite a bit, might make sense to pull it into a static method.